release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -406,19 +406,19 @@ For more information, refer to xref:../networking/openshift_sdn/assigning-egress
 ==== Network policy supports selecting host network Ingress Controllers
 
 When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
-In a network policy rule, the `policy-group.network.openshift.io/ingress=""` namespace selector label matches traffic from an Ingress Controller. You can continue to use the `network.openshift.io/policy-group: ingress` namespace selector label, but this is a legacy label that can be removed in a future release of {product-title}.
+In a network policy rule, the `policy-group.network.openshift.io/ingress:""` namespace selector label matches traffic from an Ingress Controller. You can continue to use the `network.openshift.io/policy-group: ingress` namespace selector label, but this is a legacy label that can be removed in a future release of {product-title}.
```

Comment:
@ctauchen afaik a space is permissible here.

Reply:
Searched the docs, `: ""` is clearly more common. Changed. Thanks for spotting that!